sonarts-core/docs/rules/no-nested-template-literals.md

```diff
@@ -0,0 +1,24 @@
+# no-nested-template-literals
+
+Template literals (previously named "template strings") are an elegant way to build a string without using the "+" operator to make strings
+concatenation more readable. It's possible to build complex string literals by nesting together multiple template literals. In such situation, the
+code is finally less readable and easy to maintain than without using string literals.
+
+## Noncompliant Code Example
+
+```typescript
+let color = "red";
+let count = 3;
+let message = `I have ${color ? `${count} ${color}` : count} apples`; // Noncompliant; nested template strings not easy to read
+```
+## Compliant Solution
+
+```typescript
+let color = "red";
+let count = 3;
+
```

Comment:
@andrea-guarino-sonarsource I removed this empty lines in RSPEC, could you update it here as well?